app/views/my_facilities/drug_stocks/_drug_stocks_table.html.erb

```diff
@@ -39,7 +39,6 @@
           </th>
         <% end %>
       <% end %>
-      <th class="mobile sticky"></th>
```

Comment:
this is an empty class that was showing a strange empty cell that was clickable and re-sorted the columns (like pressing the facility cell) - I've removed it